com.unity.multiplayer.mlapi/Runtime/Transports/Transport.cs

```diff
@@ -73,39 +73,33 @@ public TransportChannel[] MLAPI_CHANNELS
         public const byte MLAPI_STDRPC_CHANNEL = 1;
         public const byte MLAPI_TIME_SYNC_CHANNEL = 2;
+        public const byte MLAPI_DEFAULT_MESSAGE_CHANNEL = 3;
+        public const byte MLAPI_POSITION_UPDATE_CHANNEL = 4;
+        public const byte MLAPI_ANIMATION_UPDATE_CHANNEL = 5;
+        public const byte MLAPI_NAV_AGENT_STATE_CHANNEL= 6;
+        public const byte MLAPI_NAV_AGENT_CORRECTION_CHANNEL = 7;
 
         public static string GetChannelString(byte channel)
         {
             string channelName = "";
-            if (!TransportChannel.ChannelByteToString.TryGetValue(channel, out channelName))
-            {
-                channelName = "MLAPI_INTERNAL";
-            }
-
+            TransportChannel.ChannelByteToString.TryGetValue(channel, out channelName);
             return channelName;
         }
 
-        public static byte GetChannelByte(string channelName)
-        {
-            byte channel = MLAPI_INTERNAL_CHANNEL;
-            TransportChannel.ChannelStringToByte.TryGetValue(channelName, out channel);
-            return channel;
-        }
-
-
         /// <summary>
         /// The channels the MLAPI will use when sending internal messages.
+        /// The string for each channel is only used by the profiler
         /// </summary>
         private TransportChannel[] MLAPI_INTERNAL_CHANNELS = new TransportChannel[]
         {
-            new TransportChannel("MLAPI_INTERNAL", ChannelType.ReliableFragmentedSequenced, MLAPI_INTERNAL_CHANNEL),
+            new TransportChannel("INTERNAL", ChannelType.ReliableFragmentedSequenced, MLAPI_INTERNAL_CHANNEL),
             new TransportChannel("STDRPC", ChannelType.ReliableSequenced, MLAPI_STDRPC_CHANNEL),
-            new TransportChannel("MLAPI_TIME_SYNC", ChannelType.Unreliable, MLAPI_TIME_SYNC_CHANNEL),
-            new TransportChannel("MLAPI_DEFAULT_MESSAGE", ChannelType.Reliable, 3),
-            new TransportChannel("MLAPI_POSITION_UPDATE", ChannelType.UnreliableSequenced, 4),
-            new TransportChannel("MLAPI_ANIMATION_UPDATE", ChannelType.ReliableSequenced,5 ),
-            new TransportChannel("MLAPI_NAV_AGENT_STATE", ChannelType.ReliableSequenced, 6),
-            new TransportChannel("MLAPI_NAV_AGENT_CORRECTION", ChannelType.UnreliableSequenced, 7),
+            new TransportChannel("TIME_SYNC", ChannelType.Unreliable, MLAPI_TIME_SYNC_CHANNEL),
+            new TransportChannel("DEFAULT_MESSAGE", ChannelType.Reliable, MLAPI_DEFAULT_MESSAGE_CHANNEL),
+            new TransportChannel("POSITION_UPDATE", ChannelType.UnreliableSequenced, MLAPI_POSITION_UPDATE_CHANNEL),
+            new TransportChannel("ANIMATION_UPDATE", ChannelType.ReliableSequenced,MLAPI_ANIMATION_UPDATE_CHANNEL ),
+            new TransportChannel("NAV_AGENT_STATE", ChannelType.ReliableSequenced,MLAPI_NAV_AGENT_STATE_CHANNEL),
+            new TransportChannel("NAV_AGENT_CORRECTION", ChannelType.UnreliableSequenced, MLAPI_NAV_AGENT_CORRECTION_CHANNEL),
```

Comment:
I like const variables instead of hardcoded strings <3

minor suggestion: you could also use `nameof` operator for strings here (which would get rid of other hardcoded string here in this context as well) — for example:
`new TransportChannel(nameof(MLAPI_INTERNAL_CHANNEL), ChannelType.ReliableFragmentedSequenced, MLAPI_INTERNAL_CHANNEL)`

Reply:
oh, yeah, I'm doing that...stand by

Reply:
Actually I came up with something I think is yet cleaner and water-tight...what do you think?